ce_expansion/atomgraph/bcm.py

```diff
@@ -37,7 +37,7 @@ def recursive_update(d: dict, u: dict) -> dict:
 
 class BCModel:
     def __init__(self, atoms: ase.Atoms, metal_types: Optional[Iterable] = None,
-                 bond_list: Optional[Iterable] = None):
+                 bond_list: Optional[Iterable] = None, info: dict = {}):
         """
```

Comment:
Hey, @loevlie  - glad to see you're improving the ce_expansion code! One recommendation regarding the info dict initialization: I'd set the kwarg to `info: Optional[dict] = None`. Then in the `__init__` method, you can add the following:

```python
    if info is None:
        info = {}
```

It seems like a minor change, but you have to watch out for defaulting kwargs to mutable objects. If you init a bcm object and then change its info attribute, you might find that the next initialized bcm object has the same changes.

Hope all is well!

Michael